### PR TITLE
Add all sklearn scorers

### LINKED ISSUE #2018: [FEAT] Add sklearn metrics into the dicts at scorers.py

### PR DESCRIPTION
**Pull request by noamzbr**
#### Reference Issues/PRs
Closes #2018 

#### What does this implement/fix? Explain your changes.
Explicitly support all metrics listed in https://scikit-learn.org/stable/modules/model_evaluation.html

### CODE REVIEW

**Review by Nadav-Barak — APPROVED** [review]
Consider adding roc auc averaging to metric guide if not already there

**Review by shir22 — APPROVED** [review]
